unison-src/transcripts/tab-completion.md

```diff
@@ -50,3 +50,9 @@ unique type subnamespace.AType = A | B
 .> debug.tab-complete cd subnamespace
 .> debug.tab-complete cd subnamespace.
 ```
+
+## Tab complete against Share
+
+```ucm
+.> debug.tab-complete pull unison.pub
```

Comment:
Here's the transcript I'm thinking of just removing, since it does a web-request in the transcript.

Reply:
I think we might collect more of these "transcripts we want to run but don't want to run", maybe we should collect them in some directory and figure out what to do with them later, rather than just deleting? But no strong feelings.

Reply:
Okay, I moved this one into `transcripts-manual` and added a note there that they need to be run manually.